api/src/pcapi/core/bookings/api.py

```diff
@@ -525,6 +527,17 @@ def recompute_dnBookedQuantity(stock_ids: list[int]) -> None:
     db.session.execute(query, {"stock_ids": tuple(stock_ids)})
 
 
+def _logs_for_data_purpose(collective_bookings_subquery: Query) -> None:
+    bookings_information_tuple = CollectiveBooking.query.filter(
+        CollectiveBooking.id.in_(collective_bookings_subquery)
```

Comment:
pourquoi un in ici ? pourquoi ne pas faire directement un withentities sur la query ? (ça ne marche peut être pas, je poe juste la question)